tutorials/files/underwater_vehicles/my_lrauv/model.sdf

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0"?>
+<sdf version="1.6">
+  <model name="my_lrauv">
+    <!-- Body -->
+    <link name="base_link">
+      <inertial>
+        <mass>147.8671</mass>
+        <inertia>
+          <ixx>3.000000</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>41.980233</iyy>
+          <iyz>0</iyz>
+          <izz>41.980233</izz>
+        </inertia>
+      </inertial>
+      <!-- Uncomment this block to enable neutral buoyancy. -->
+      <collision name="main_body_buoyancy">
+        <geometry>
+          <box>
+            <size>2 0.3 0.2464451666666667</size>
+          </box>
+        </geometry>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh>
+            <uri>meshes/tethys.dae</uri>
+            <submesh>
+              <name>Body</name>
+                <center>false</center>
+            </submesh>
+          </mesh>
+        </geometry>
+        <material>
+          <diffuse>1.0 1.0 1.0</diffuse>
+          <specular>1.0 1.0 1.0</specular>
+          <pbr>
+            <metal>
+              <albedo_map>materials/textures/Tethys_Albedo.png</albedo_map>
+              <normal_map>materials/textures/Tethys_Normal.png</normal_map>
+              <metalness_map>materials/textures/Tethys_Metalness.png</metalness_map>
+              <roughness_map>materials/textures/Tethys_Roughness.png</roughness_map>
+            </metal>
+          </pbr>
+          <script>
+            <uri>materials/scripts/</uri>
+            <uri>materials/textures/</uri>
+            <name>tethys/Tethys_Diffuse</name>
+          </script>
```

Comment:
this is used by gazebo-classic. I think we can remove these lines, as well as the material script `underwater_vehicles/my_lrauv/materials/scripts/model.material`